app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -62,11 +62,13 @@ function DocumentCaptureReviewIssues({
         altFailedDocTypeMsg={isFailedDocType ? t('doc_auth.errors.doc.wrong_id_type_html') : null}
         hasDismissed={hasDismissed}
       />
-      <p>
-        <HtmlTextWithStrongNoWrap
-          text={t('idv.failure.attempts_html', { count: remainingSubmitAttempts })}
-        />
-      </p>
+      {remainingSubmitAttempts !== Infinity && (
```

Comment:
Might be a good use case for [`Number.isFinite`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite) ?

```suggestion
      {Number.isFinite(remainingSubmitAttempts) && (
```

Reply:
I was thinking of it more in practical terms of "We want to show the message when we have a finite number to work with" and having a built-in function that supports that, not targeting specific values we want to exclude.